lib/reporters/junit/report.xml

```diff
@@ -1,6 +1,6 @@
 <testsuite tests="{{this.count}}" skipped="{{this.skipped}}" errors="{{this.errors}}" failures="{{this.failures}}" time="{{this.time}}">
     {{#each this.tests~}}
-    <testcase classname="{{this.title}}" name="{{this.title}}" time="{{this.time}}">
+    <testcase classname="{{this.path}}" name="{{this.relativeTitle}}" time="{{this.time}}">
```

Comment:
Why should the name change to the `relativeTitle`? If we add more wrapping testsuite elements for each of the experiments then maybe we can reduce the name for each testcase.

Reply:
relativeTitle is the test case name. As per junit standards we can have classname as suite name and name as testcase name.